install/rhel-installation-guide.md

```diff
@@ -0,0 +1,32 @@
+# Installing Kata Containers on RHEL
```

Comment:
Suggest removing the gerund for consistency: Install Kata Containers on RHEL

Reply:
Hi @klynnrif - just so you're aware, @GabyCT has made these files as close as possible to the existing install guides as requested. So if we apply this change, @GabyCT is going to have to also update:

- https://github.com/kata-containers/documentation/blob/master/install/centos-installation-guide.md
- https://github.com/kata-containers/documentation/blob/master/install/fedora-installation-guide.md
- https://github.com/kata-containers/documentation/blob/master/install/ubuntu-installation-guide.md

Reply:
Hi @jodh-intel and @GabyCT - that is what I thought. I can go back in and approve w/o changes for this one if you prefer. Let me know. Thanks!

Reply:
I think the changes you're suggesting make a lot of sense so if @GabyCT is happy to do the changes...? :)

Since these documents are actually "executed" now, I'm really keen on keeping them as similar as possible.

Reply:
@jodh-intel @GabyCT Perfect! Makes me happy :)